src/pages/product_listing/product.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Card from '../../components/Card'
+import Footer from '../../components/Footer'
+import Nav from '../../components/Nav'
```

Comment:
Suggestion: Similarly as said above for pages, you can export each component from `index.js` file inside components